### DISCUSSION TITLE
API endpoint not accessible from public internet

### DISCUSSION THREAD

**Discussion by sumit-rawat-quo** [thinkarlabs/P23-004-MalnutritionAwareness#1] [General]
Hi @srivatsapraveen ,
I installed nginx & other dependencies following this [link](https://www.travisluong.com/how-to-deploy-fastapi-with-nginx-and-supervisor/). 
I have written a basic Fast API hello world API and I am trying to access it through public internet. However I am unable to connect through http://13.71.69.219 , it seems like VM doesn't have NSG for accessing through public internet and credentials through which I logged in doesn't allow me to add/edit Network Security Groups. 
Can you please help adding a NSG for port 80 from source anywhere so that we could access the API endpoint ?
cc: @deepak-quovantis 
Thanks

**Comment by srivatsapraveen** [comment]
Hi @sumit-rawat-quo - NSG (network security groups) are not required for public VM access. All it needs is a rule for inbound ports (80, 443, other than 22 for ssh). To confirm, I setup another VM at http://[104.211.202.1](http://104.211.202.1/)/ and it returns the "Hello from Azure VM World" message from FastAPI. (Note : inbound port rules for 80 and 443 are enable for the VM-P23-004-MalnutritionAwareness VM too)

![image](https://user-images.githubusercontent.com/4785801/215523735-aa68f792-a7f0-461d-a84a-f47f6d5b7308.png)
